src/components/SignalCard/SignalCard.js

```diff
@@ -149,4 +118,48 @@ const SignalCardBottom = ({
   )
 }
 
+const MoreSignalActions = ({ signalId, removeSignal }) => {
+  const [isOpen, setOpen] = useState(false)
+
+  return (
+    <div className={styles.more} onMouseLeave={() => setOpen(false)}>
+      <Button
+        onClick={() => setOpen(!isOpen)}
+        className={cx(styles.expandButton)}
+      >
+        <Icon type='dots' className={styles.moreIcon} />
+      </Button>
```

Comment:
looks like [`ContextMenu` component](https://storybook.stage.internal.santiment.net/?selectedKind=ContextMenu&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)
example in app: 
![image](https://user-images.githubusercontent.com/24521041/60617061-88cc4d80-9ddb-11e9-9b2e-355c414b0259.png)

Reply:
Correct, will check